src/codegen/generate.ts

```diff
@@ -588,6 +589,12 @@ export default class ApiGenerator {
       // items -> array
       return factory.createArrayTypeNode(this.getTypeFromSchema(schema.items));
     }
+    if ("prefixItems" in schema && schema.prefixItems) {
+      // prefixItems -> typed tuple
+      return factory.createTupleTypeNode(
+        schema.prefixItems.map((schema) => this.getTypeFromSchema(schema)),
+      );
+    }
```

Comment:
Shouldn't this check `maxItems` and `items` so that if `prefixItems` is truly used to only describe a prefix, it becomes an open-ended tuple like `[...TPrefixItems, ...TItems]`?

Reply:
To be honest, I didnt really understand what `prefixItems` are supposed to  do (just checked [here](https://json-schema.org/understanding-json-schema/reference/array?highlight=prefixitems#additional-items) but I think you're right, the result should add potential additional items. Currently, `items` and `prefixItems` cannot be used together. I doubt it's too hard to do though?

Reply:
Should we treat this as a bug? Is there an issue for this?

Reply:
For context, I'm just getting back from a 6weeks time-off and don't have the time to understand in detail what this is about. If this is something that will affect others, can one of you please open an issue?